lib/cli.go

```diff
@@ -164,7 +165,7 @@ func run(c *cli.Context) error {
 	}()
 
 	if err := obj.Run(); err != nil {
-		//return cli.NewExitError(err.Error(), 1) // TODO: ?
+		log.Printf("%s: %s", reflect.TypeOf(obj).Elem().String(), err.Error())
```

Comment:
What is this use of `reflect` about?

Reply:
It gets the name of the object the error originated from

Reply:
i looked in some other places in that file and settled on 'log.Printf("Main: Error: %v", err)'